Game crashes when loading a singleplayer world

When using the Minecraft 1.21.10 version of Plasmid (69d9558eda611d4abe1615fa7b47fdabce4c95ac), the game crashes immediately after trying to create a new singleplayer world or load an existing one:

```
java.lang.RuntimeException: Mixin transformation of net.minecraft.server.integrated.IntegratedPlayerManager failed
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Callback method canSavePlayerData(ZLnet/minecraft/server/network/ServerPlayerEntity;)Z in plasmid.client.mixins.json:game.space.IntegratedPlayerManagerMixin from mod plasmid failed injection check, (0/1) succeeded. Scanned 0 target(s). No refMap loaded.
```

Mixin verification tests would be useful for avoiding this class of issue.

--- a/src/client/java/xyz/nucleoid/plasmid/client/mixin/game/space/IntegratedPlayerManagerMixin.java
+++ b/src/client/java/xyz/nucleoid/plasmid/client/mixin/game/space/IntegratedPlayerManagerMixin.java
@@ -12,7 +12,7 @@ public abstract class IntegratedPlayerManagerMixin {
 
     @ModifyExpressionValue(
             method = "savePlayerData",
-            at = @At(value = "INVOKE", target = "Lnet/minecraft/server/integrated/IntegratedServer;isHost(Lcom/mojang/authlib/GameProfile;)Z")
+            at = @At(value = "INVOKE", target = "Lnet/minecraft/server/integrated/IntegratedServer;isHost(Lnet/minecraft/server/PlayerConfigEntry;)Z")
     )
     private boolean canSavePlayerData(boolean original, ServerPlayerEntity player) {
         return original && !GameSpaceManagerImpl.get().inGame(player);

